### PR TITLE
Add loadYtyp & unloadYtyp

### PR DESCRIPTION
**Pull request by C0kkie**

### CODE REVIEW

File: client/index.d.ts [hunk 1]
```diff
@@ -1885,4 +1885,24 @@ declare module "alt-client" {
...
+
+  /**
+   * Load a specific ytyp file.
+   *
+   * @param path Relative path to the game folder
+   * @remarks Example: alt.loadYtyp("x64u.rpf/levels/gta5/_hills/country_06/country_06_metadata.rpf/cs6_08_interior_cs6_08_mine_int.ytyp")
+   *
+   * @alpha
+   */
+  export function loadYtyp(path: string): void;
```
Comment (LeonMrBonnie) [inline client/index.d.ts:1897]: The function actually returns `boolean` instead of `void`.

File: client/index.d.ts [hunk 2]
```diff
@@ -1885,4 +1885,24 @@ declare module "alt-client" {
...
+
+  /**
+   * Unload a specific ytyp file.
+   *
+   * @param path Relative path to the game folder
+   * @remarks Example: alt.unloadYtyp("x64u.rpf/levels/gta5/_hills/country_06/country_06_metadata.rpf/cs6_08_interior_cs6_08_mine_int.ytyp")
+   *
+   * @alpha
+   */
+   export function unloadYtyp(path: string): void;
```
Comment (LeonMrBonnie) [inline client/index.d.ts:1907]: The function actually returns `boolean` instead of `void`.

File: client/index.d.ts (hunk 1 above)
Comment (C0kkie) [reply client/index.d.ts]: fixed

File: client/index.d.ts (hunk 2 above)
Comment (C0kkie) [reply client/index.d.ts]: fixed

**Comment by Lhoerion** [comment]
You don't put examples in remarks tag, you've special tag for it
https://jsdoc.app/tags-example.html

**Comment by C0kkie** [comment]
Should be fixed here https://github.com/altmp/altv-types/pull/41/files
@Lhoerion

**Pull request merged**